### ISSUE TITLE
Only the first wishlist is clickable in humingbird theme

### ISSUE THREAD

**Issue by leemyongpakva** [PrestaShop/PrestaShop#35365]
### Prerequisites

- [X] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md).
- [X] I have already [searched in existing issues](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3ABug) and found no previous report of this bug.

### Describe the bug and add attachments

The problem happens in hummingbird theme with **more than one wishlist**.
In the first load the second, the third or so on are clickable, but once you hover them, they will become un-clickable till the next refresh.

### Expected behavior

All the wishlists are clickable whether they are hovered over or not.

### Steps to reproduce

1. Get a PS 8.x site with hummingbird theme and blockwishlist 3.0.1 active
2. Login and create a wishlists with **at least two** wishlist
3. Navigate to Your account > Wishlist, move mouse cursor over the first wishlist and the second wishlist, see the second one become unclickable.

### PrestaShop version(s) where the bug happened

8.1

### PHP version(s) where the bug happened

7.4

### If your bug is related to a module, specify its name and its version

blockwishlist 3.0.1

**Comment by AureRita** [comment]
Hi @leemyongpakva ,

Thank you for your report, I reproduce the issue with PrestaShop version develop, as you can see : 

[recording(100).webm](https://github.com/PrestaShop/PrestaShop/assets/100699445/28e1bacd-caa3-46a5-b020-916c8681f673)

Currenlty, this issue seems similar to this one : 
https://github.com/PrestaShop/PrestaShop/issues/35276

Are you agree with that ? 

Waiting for your feedback

**Comment by SharakPL** [comment]
@AureRita yes, it's related. Those hidden buttons are inside modals. My PR will fix that too.

**Comment by leemyongpakva** [comment]
@AureRita Yes, they are related and derived from the same hidden modals, but not duplicated. The first one is about hidden buttons after deleting one list. The latter is about unclickable list after hovering over.
@SharakPL I think separating Product button issue to another issue will be better for both Review and QA.
